sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -230,6 +236,76 @@ describe("Streaming - Misc Tests", function(): void {
     );
     await testManualComplete();
   });
+
+  class TestTokenProvider extends SasTokenProvider {
```

Comment:
Since this test token provider is very specific to our test case where we want the first attempt to fail, it is not re-usable by other tests. Please move it inside the test case that was added to avoid potential confusion around its usage in the guture